setup.py

```diff
@@ -39,6 +39,7 @@ def get_c_sources(include_headers=False):
     # HACK.
     # See https://github.com/pypa/packaging-problems/issues/84.
     sources=get_c_sources(include_headers=("sdist" in sys.argv)),
+    extra_compile_args=["-static", "-static-libgcc", "-static-libstdc++"],
```

Comment:
Should this really be done even on Linux? I don't quite understand how it works; why is compiling with -static and linking without it not an error?

Reply:
Advantage of statically linking on Linux: It would be possible to deploy a binary wheel on thin servers without gcc installed (uploading to PyPI is impossible though).

Disadvantage: security updates to the gcc lib will of course not be applied without recompiling.

Since the `-static` link flag seems to be required on Windows (it works without it, but will silently dynamically link), perhaps it should always be added to the extra link args? From what I read, the compile flags should have been enough.

Reply:
Can definitely drop this on Linux for now.

Reply:
Either way (dropping on Linux or adding -static to link on Linux) is fine with me.

Reply:
I tested that, just adding `-static` to `extra_link_args` on Windows, then it not dynamically link, `extra_compile_args` is not required.

Reply:
Let's do that then,